src/api/rest/server/middlewares/zerologger.go

```diff
@@ -42,33 +42,35 @@ func Zerologger(skipPatterns [][]string) echo.MiddlewareFunc {
 		// HandleError:      true, // forwards error to the global error handler, so it can decide appropriate status code
 		LogValuesFunc: func(c echo.Context, v middleware.RequestLoggerValues) error {
 			if v.Error == nil {
-				log.Info().
-					Str("id", v.RequestID).
-					Str("client_ip", v.RemoteIP).
-					//Str("host", v.Host).
-					Str("method", v.Method).
-					Str("URI", v.URI).
-					//Str("user_agent", v.UserAgent).
-					Int("status", v.Status).
-					//Int64("latency", v.Latency.Nanoseconds()).
-					Str("latency_human", v.Latency.String()).
-					Str("bytes_in", v.ContentLength).
-					Int64("bytes_out", v.ResponseSize).
-					Msg("request")
+				if v.Method != "OPTIONS" {
```

Comment:
다음 const를 활용하시면 좋을 것 같습니다. `"net/http"` pkg import 필요
: 확인해보니, echo.OPTIONS는 deprecated되었습니다... (https://pkg.go.dev/github.com/labstack/echo/v4#pkg-constants)
```suggestion
				if v.Method != http.MethodOptions {
```

---
(확인 차 답글을 남겨 놓는 부분입니다.)
1. Skipper에서 처리하지 않으셨길래 생각해본 바로는, 존재하지 않는 API 요청(Error)에 대해서는 로그를 남기시려는 의도로 이해됩니다.

2. 다음 코드를 보니 이후에 조건이 추가되는 것을 고려하고 계신 것 같습니다. :-)
```
if v.Error == nil
  if v.Method != "OPTIONS"
```

Reply:
네네. 일단 OPTIONS라도 error라면 출력되는 것을 고려하였습니다.